### PR TITLE
GATEWAY-2876: Added fix for CXF timestamp validation.  Exposed CXF times...

### PR DESCRIPTION
**Pull request by akong**
...tamp configuration.

### CODE REVIEW

File: Product/Production/Gateway/AdminDistribution_10/src/main/resources/CONNECT-interceptor-beans.xml [hunk 1]
```diff
@@ -5,5 +5,6 @@
```
Comment (bhumphrey) [inline Product/Production/Gateway/AdminDistribution_10/src/main/resources/CONNECT-interceptor-beans.xml:-1]: CONNECT-interceptor-bean.xml can be centralized to CONNECTCoreLib

File: Product/SoapUI_Test/ValidationSuite/src/test/resources/domain1/config/domain.xml [hunk 2]
```diff
@@ -317,6 +317,7 @@
         <jvm-options>-Dcom.sun.enterprise.security.httpsOutboundKeyAlias=gateway</jvm-options>
         <jvm-options>-DSERVER_KEY_ALIAS=gateway</jvm-options>
         <jvm-options>-DCLIENT_KEY_ALIAS=gateway</jvm-options>
+        <jvm-options>-Dcom.sun.grizzly.writeTimeout=30000</jvm-options>
```
Comment (bhumphrey) [inline Product/SoapUI_Test/ValidationSuite/src/test/resources/domain1/config/domain.xml:320]: what about websphere and weblogic?

File: Product/Production/Gateway/AdminDistribution_10/src/main/resources/CONNECT-interceptor-beans.xml (hunk 1 above)
Comment (akong) [inline Product/Production/Gateway/AdminDistribution_10/src/main/resources/CONNECT-interceptor-beans.xml:-1]: Ok.  I'll see what I need to do to consolidate all of it to CoreLib.

File: Product/SoapUI_Test/ValidationSuite/src/test/resources/domain1/config/domain.xml (hunk 2 above)
Comment (akong) [inline Product/SoapUI_Test/ValidationSuite/src/test/resources/domain1/config/domain.xml:320]: The configuration above is glassfish only.  I felt that it was beyond the scope of this effort (i.e. trying to find all the configuration that may affect large payloads for all servers).  I am updating the wiki to provide a guide to configure glassfish which they can use as an aid for configuring websphere/weblogic.  With that said, I believe we are doing performance testing with webpshere/weblogic and large payloads.  We can update then the instructions/config files if we see anything that will alleviate whatever bottlenecks that may pop up.

**Pull request merged**